src/components/SearchGridFilter.vue

```diff
@@ -92,41 +169,114 @@ export default {
+      return check;
+    },
   },
 };
 </script>
 
 <style lang="scss" scoped>
 
+.close {
+  float: right;
+  background: none;
+}
+
+.modal {
+  width: 500px;
+  max-height: 600px;
+  margin: 0px auto;
+  background-color: #F5F5F5;
+  border-radius: 2px;
+  overflow-y: scroll;
+  box-shadow: 0 2px 8px 3px;
+  transition: all 0.2s ease-in;
+  color: #000000;
+}
+
+.fadeIn-enter {
+  opacity: 0;
+}
+
+.fadeIn-leave-active {
+  opacity: 0;
+  transition: all 0.2s step-end;
+}
+
+.fadeIn-enter .modal,
+.fadeIn-leave-active.modal {
+  transform: scale(1.1);
+}
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  background: #00000094;
+  z-index: 999;
+  transition: opacity 0.2s ease;
+}
+
+h4 {
+  text-transform: none;
+}
+
 .search-filters {
   display: none;
   height: auto;
   top: 0;
   position: sticky;
-
   label {
     color: #333333;
   }
-
   &__visible {
     border-top: 1px solid #e8e8e8;
     display: block;
   }
 }
 
+.is-flex {
```

Comment:
`.is-flex` is a bulma class. You shouldn't add to its styles. Please change this class name

Reply:
Done :)